app/ide-desktop/lib/dashboard/src/authentication/src/providers/logger.tsx

```diff
@@ -0,0 +1,52 @@
+/** @file Defines the React provider for the {@link Logger} interface, along with a hook to use the
+ * provider via the shared React context. */
+import * as react from "react";
+
+// ==============
+// === Logger ===
+// ==============
+
+/** Interface used to log logs, errors, etc.
+ *
+ * In the browser, this is the `Console` interface. In Electron, this is the `Logger` interface
+ * provided by the EnsoGL packager. */
+export interface Logger {
+  /** Logs a message to the console. */
+  log: (message?: any, ...optionalParams: any[]) => void;
```

Comment:
prefer `unknown` over `any`

Reply:
also the first parameter should probably not be optional

Reply:
IIRC this is the API as it appears on the actual `Console` implementation. I suppose there's no harm in us making the API stricter than it is.

Reply:
yeah, it is indeed the actual `Console` API. note though that this isn't any stricter, just less unsafe.
accepting `unknown`, `...unknown[]` accepts literally anything (except for passing zero arguments) anyway

i suspect the reason console is using `any` is because its typings were probably written before `unknown` existed